public/docs/v1/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
Are creating some documentation? If not you do not need this file...

Reply:
No, it must have been added by versionist.